packages-backend/express/src/auth.ts

```diff
@@ -149,9 +155,15 @@ function createSessionAuthVerifier<
       issuer = mapToLegacyIssuer(cloudIdentifierHeader) ?? issuer;
     }
 
-    // @ts-ignore: the node HTTP request does not know about `originalUrl`
-    const requestUrlPath = request.originalUrl ?? request.url;
-    const audience = getConfiguredAudience(options, requestUrlPath);
+    const requestUrlPath = options.getRequestUrl
+      ? options.getRequestUrl(request)
+      : request.originalUrl ?? request.url;
+
+    if (!requestUrlPath || requestUrlPath === 'undefined') {
+      throw new Error('Could not get the request URL.');
```

Comment:
Not sure if this message is enough or we should be more verbose (maybe a link to the docs?).